Protacon.RxMq.AzureServiceBus/Topic/AzureTopicPublisher.cs

```diff
@@ -86,14 +86,99 @@ public void Dispose()
                 .ToList()
                 .ForEach(x => x.Dispose());
         }
+
         public Task SendAsync<T>(T message) where T : new()
         {
             var topic = _settings.TopicNameBuilder(message.GetType());
 
             if (!_bindings.ContainsKey(topic))
-                _bindings.Add(topic, new Binding(_settings, _topicManagement, topic, typeof(T), _logger));
+                return TryCreateBinding(topic, typeof(T), message, 5, 60);
+            
+            return _bindings[topic].SendAsync(message);
+        }
+
+        /// <summary>
+        /// Creates new Binding for topic in a safe way. If initial tries fail, it will fallback to intervalled retrys.
+        /// </summary>
+        /// <param name="topic"></param>
+        /// <param name="type"></param>
+        /// <param name="message"></param>
+        /// <param name="instantRecoveryTries"></param>
+        /// <param name="lifeCycleRecoveryInterval"></param>
+        private Task TryCreateBinding(string topic, Type type, object message, int instantRecoveryTries, int lifeCycleRecoveryInterval)
+        {
+            CancellationTokenSource cancellation = new CancellationTokenSource();
+            int lifeCycleTryCount = 0;
+
+            for (var i = 1; i <= instantRecoveryTries; i++)
+            {
+                _logger.LogInformation(
+                    $"TryCreateBinding: Try {i} of {instantRecoveryTries} for binding ('{topic}')");
```

Comment:
You could also use `{nameof(TryCreateBinding)}` to protect this from renaming.

Reply:
That's true, changed